src/pybind/mgr/dashboard/frontend/src/app/core/navigation/breadcrumbs/breadcrumbs.component.spec.ts

```diff
@@ -128,4 +136,36 @@ describe('BreadcrumbsComponent', () => {
     component.ngOnDestroy();
     expect(component.subscription.closed).toBeTruthy();
   });
+
+  it('should display no breadcrumbs in page title when navigating to dashboard', fakeAsync(() => {
+    fixture.ngZone.run(() => {
+      router.navigateByUrl('');
+    });
+    tick();
+    expect(titleService.getTitle()).toEqual('Ceph');
+  }));
+
+  it('should display no breadcrumbs in page title when a page is not found', fakeAsync(() => {
+    fixture.ngZone.run(() => {
+      router.navigateByUrl('/error');
+    });
+    tick();
+    expect(titleService.getTitle()).toEqual('Ceph');
+  }));
+
+  it('should display 2 breadcrumbs in page title when navigating to hosts', fakeAsync(() => {
+    fixture.ngZone.run(() => {
+      router.navigateByUrl('/hosts');
+    });
+    tick();
+    expect(titleService.getTitle()).toEqual('Ceph: Cluster > Hosts');
+  }));
+
+  it('should display 3 breadcrumbs in page title when navigating to RBD Add', fakeAsync(() => {
+    fixture.ngZone.run(() => {
+      router.navigateByUrl('/block/rbd/add');
+    });
+    tick();
+    expect(titleService.getTitle()).toEqual('Ceph: Block > Images > Add');
```

Comment:
This is great. I wonder if we could now remove (or at least revisit) the end-2-end tests that check the breadcrumbs (and which randomly fail quite often :/)...